tests/user_metadata_test.go

```diff
@@ -0,0 +1,130 @@
+package tests
+
+import (
+	"github.com/pborman/uuid"
+	commonpb "go.temporal.io/api/common/v1"
+	enumspb "go.temporal.io/api/enums/v1"
+	sdkpb "go.temporal.io/api/sdk/v1"
+	taskqueuepb "go.temporal.io/api/taskqueue/v1"
+	updatepb "go.temporal.io/api/update/v1"
+	"go.temporal.io/api/workflowservice/v1"
+)
+
+func getDescribeWorkflowExecutionInfo(engine FrontendClient, namespace string, workflowID string, runID string) (*workflowservice.DescribeWorkflowExecutionResponse, error) {
+	return engine.DescribeWorkflowExecution(NewContext(), &workflowservice.DescribeWorkflowExecutionRequest{
+		Namespace: namespace,
+		Execution: &commonpb.WorkflowExecution{
+			WorkflowId: workflowID,
+			RunId:      runID,
+		},
+	})
+}
+
+func prepareTestUserMetadata() *sdkpb.UserMetadata {
```

Comment:
This is not the first time I've seen test helpers above the test cases, but I'm not sure why that would be beneficial? Isn't the most interesting bit the test case and shouldn't that come first?

Reply:
Agree. Will move it down.